python/pyspark/sql/tests.py

```diff
@@ -6394,6 +6394,17 @@ def test_invalid_args(self):
                 df.withColumn('mean_v', mean_udf(df['v']).over(ow))
 
 
+class DataSourceV2Tests(ReusedSQLTestCase):
+    def test_pyspark_udf_SPARK_25213(self):
+        from pyspark.sql.functions import udf
+
+        df = self.spark.read.format("org.apache.spark.sql.sources.v2.SimpleDataSourceV2").load()
```

Comment:
I think this test will fail if test classes are not compiled. Can we check if test classes are compiled and then skip if not existent?